docs/user/installation_and_configuration.rst

```diff
@@ -32,12 +32,14 @@ Versions and Downloads
 .. note::
 
     The current recommended version of GeoMesa to install is |release|.
+    For Kafka 09 and Kafka 10 download and build the source release. (:ref:`building_from_source`)
 
 **Latest release**: |release|
 
 .. TODO: substitutions don't work in some kinds of markup, including URLs
 
-* Release tarball: |release_tarball|
+* Accumulo release tarball: |release_tarball|
```

Comment:
This got renamed to ``|release_tarball_accumulo|``:

``[exec] /opt/devel/src/geomesa/docs/user/installation_and_configuration.rst:41: ERROR: Undefined substitution referenced: "release_tarball"``